sql/moz-fx-data-shared-prod/firefox_desktop_derived/urlbar_events_daily_v1/query.sql

```diff
@@ -0,0 +1,110 @@
+WITH
+  temp_unnested AS (
+  SELECT
+    submission_date,
+    event_id,
+    event_action,
+    res.product_result_type AS product_result_type,
+    normalized_channel,
+    normalized_country_code,
+    pref_fx_suggestions,
+    pref_sponsored_suggestions,
+    is_terminal,
+    CASE
+      WHEN (product_selected_result = res.product_result_type AND event_action = 'engaged' AND is_terminal = TRUE) THEN 1
+    ELSE
+    0
+  END
+    AS is_clicked,
+    CASE
+      WHEN (product_selected_result = res.product_result_type AND event_action = 'annoyance') THEN 1
+    ELSE
+    0
+  END
+    AS is_annoyed,
+  FROM
+    `mozdata.firefox_desktop.urlbar_events`
+  CROSS JOIN
+    UNNEST (results) AS res
+  WHERE
+    submission_date = @submission_date ),
+  temp_session AS (
+  SELECT
+    submission_date,
+    event_id,
+    product_result_type,
+    is_terminal,
+    mozdata.udf.mode_last(ARRAY_AGG(normalized_channel)) AS normalized_channel,
+    mozdata.udf.mode_last(ARRAY_AGG(normalized_country_code)) AS normalized_country_code,
+    mozdata.udf.mode_last(ARRAY_AGG(pref_fx_suggestions)) AS pref_fx_suggestions,
+    mozdata.udf.mode_last(ARRAY_AGG(pref_sponsored_suggestions)) AS pref_sponsored_suggestions,
+    LOGICAL_OR(is_clicked > 0) AS is_clicked,
+    LOGICAL_OR(is_annoyed > 0) AS is_annoyed,
+    LOGICAL_OR(is_terminal = TRUE) AS n_impressions,
+  FROM
+    temp_unnested
+  GROUP BY
+    submission_date,
+    event_id,
+    product_result_type,
+    is_terminal ),
+  total_urlbar_sessions AS (
+  SELECT
+    submission_date,
+    normalized_country_code,
+    normalized_channel,
+    COUNT(DISTINCT event_id) AS urlbar_sessions
+  FROM
+    temp_session
+  WHERE
+    is_terminal = TRUE
+  GROUP BY
```

Comment:
This should probably group by all the grouping fields in `daily_counts` so they can be used as filters